scripts/setup.sh

```diff
@@ -2,3 +2,5 @@
 
 python -m pip install --upgrade pip
 pip install -r requirements.txt
+python -m spacy download en_core_web_trf
+pip install spacy-transformers
```

Comment:
This should be in the `requirements.txt` file. This can be done by running `pip3 freeze > requirements.tx`.

Reply:
Done